wp-content/themes/access/src/scss/alert.scss

```diff
@@ -46,4 +46,22 @@
   .c-alert-banner__body {
     margin-right: 5px;
   }
-}
\ No newline at end of file
+}
+
+@media (min--moz-device-pixel-ratio: 1.5),
```

Comment:
You're repeating the styling that's between line 34 and 49. I would replace it with this snippet you've added.